### PR TITLE
Finish API tests

### PR DESCRIPTION
**Pull request by webrouse**
Jira: https://keboola.atlassian.net/browse/COM-873

### CODE REVIEW

File: src/api/branch_test.go [hunk 1]
```diff
@@ -54,6 +56,7 @@ func TestBranchApiCalls(t *testing.T) {
 	request := a.CreateBranchRequest(branchBar).
 		OnSuccess(func(response *client.Response) *client.Response {
 			// OnSuccess callback called when job is in successful state
+			job := response.Result().(*model.Job)
```
Comment (webrouse) [inline src/api/branch_test.go:59]: pouzivala sa nespravna premenna.

File: src/api/config.go [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
-	"fmt"
 	"github.com/go-resty/resty/v2"
+	"github.com/spf13/cast"
 	"keboola-as-code/src/client"
-	"keboola-as-code/src/json"
 	"keboola-as-code/src/model"
-	"strconv"
 )
 
+func (a *StorageApi) ListComponents(branchId int) (*[]*model.Component, error) {
```
Comment (webrouse) [inline src/api/config.go:10]: Presunute do `config.go`, kedze to tiez nacitava konfiguracie -> ale pre vsetky komponenty.

File: src/api/config_test.go [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+import (
+	"github.com/stretchr/testify/assert"
+	"keboola-as-code/src/json"
+	"keboola-as-code/src/model"
+	"keboola-as-code/src/utils"
+	"testing"
+)
+
+func TestConfigApiCalls(t *testing.T) {
```
Comment (webrouse) [inline src/api/config_test.go:11]: Test pre `config` API calls.

File: src/api/row_test.go [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+import (
+	"github.com/stretchr/testify/assert"
+	"keboola-as-code/src/json"
+	"keboola-as-code/src/model"
+	"keboola-as-code/src/utils"
+	"testing"
+)
+
+func TestConfigRowApiCalls(t *testing.T) {
```
Comment (webrouse) [inline src/api/row_test.go:11]: Test pre `config row` api calls.

**Review by webrouse — COMMENTED** [review]
V tomto PR su dokoncene vsetky testy ohladom API.

**Review by ondrajodas — APPROVED** [review]

**Pull request merged**